### PR TITLE
IGAPP-883: Only log firebase errors

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
No need to report these kind of errors to sentry.

File: native/src/utils/PushNotificationsManager.ts [hunk 1]
```diff
@@ -43,10 +43,10 @@ export const unsubscribeNews = async (city: string, language: string): Promise<v
   try {
     const messaging = await importFirebaseMessaging()
     await messaging().unsubscribeFromTopic(topic)
+    log(`Unsubscribed from ${topic} topic!`)
   } catch (e) {
-    reportError(e)
+    log(`Failed to unsubscribe from ${topic} topic: ${e instanceof Error ? e.message : 'Unknown Error'}`)
```
Comment (Taggotty) [inline native/src/utils/PushNotificationsManager.ts:48]: But isn't it also a firebase error if unsubscribing fails?

**Review by SteffiStuffel — APPROVED** [review]
Why only a Draft?

**Comment by klinzo** [comment]
> Why only a Draft?

@SteffiStuffel because this basically suppresses sending all errors of firebase to sentry. Not sure what we can do apart from that though. Perhaps I'll try upgrading.

**Comment by klinzo** [comment]
Won't do, we'll just ignore this in sentry.

**Pull request closed without merging**